Marlin/src/pins/stm32f4/pins_RUMBA32_common.h

```diff
@@ -144,3 +167,10 @@
   #endif
 
 #endif
+
+// Alter timing for graphical display
+#if HAS_GRAPHICAL_LCD
+  #define BOARD_ST7920_DELAY_1 DELAY_NS(96)
+  #define BOARD_ST7920_DELAY_2 DELAY_NS(48)
+  #define BOARD_ST7920_DELAY_3 DELAY_NS(600)
+#endif
```

Comment:
This block can be within the `HAS_SPI_LCD` block above, and it can be…
```cpp
#if HAS_GRAPHICAL_LCD
  #ifndef BOARD_ST7920_DELAY_1
    #define BOARD_ST7920_DELAY_1 DELAY_NS(96)
  #endif
  #ifndef BOARD_ST7920_DELAY_2
    #define BOARD_ST7920_DELAY_2 DELAY_NS(48)
  #endif
  #ifndef BOARD_ST7920_DELAY_3
    #define BOARD_ST7920_DELAY_3 DELAY_NS(600)
  #endif
#endif
```